rule-packs/cyberark-idaptive-misconfigurations.json

```diff
@@ -53,7 +53,7 @@
     "queries": [
       {
         "name": "query0",
-        "query": "FIND cyberark_idaptive_device WITH complianceState != 'compliant' OR 'Compliant'",
+        "query": "FIND cyberark_idaptive_device WITH complianceState != ('compliant' OR 'Compliant')",
```

Comment:
Can I recommend switching this to regex to get the case insensitive match if that is needed? This also raises the question of, why is the property value inconsistent?

```
FIND cyberark_idaptive_device WITH complianceState = /compliant/i
```

Reply:
At least in j1dev this does not execute successfully
<img width="946" alt="image" src="https://github.com/JupiterOne/jupiterone-alert-rules/assets/67770587/944dd775-c8a3-42a8-9291-52d19fd1c259">

Reply:
This is a good suggestion, using regex is a great new capability in the query language for providing flexibility in expected properties.. for this case we have yet to gain access to an instance of the CyberArk tool.. could this be why the multiple expected properties were worked into the query? @mikiodehartj1